relational_database/homework.py

```diff
@@ -105,19 +118,27 @@ def task_8_count_customers_by_city(cur):
     Returns: 69 records in descending order
 
     """
-    pass
+    cur.execute('''SELECT COUNT(customerid) AS count, city FROM CUSTOMERS 
+    GROUP BY city ORDER BY city DESC;''')
+    result = cur.fetchall()
+    result.sort(key=lambda x: x['city'], reverse=True) # necessary to solve issue with non-ascii characters(in
```

Comment:
Do you really need this?

Reply:
Unfortunately, I really do! I think the issue is in OS locale settings. The control result for task 8 is unsorted, so after sorting the latter you get the order like this - from 'A' to 'z' and than non-ascii caracter like 'Å'(order number of 'Å' is 197 in UTF-8), while PostgreSQL query in my case always returns either 'Å' to 'z' or 'z' to 'Å' depending on order. With the aim of make two results comparable I did this double sorting. 
I would appreciate if you could either clarify or suggest the better way of solving the issue.